embedded-aerospike-enterprise/src/main/java/com/playtika/testcontainers/aerospike/enterprise/AerospikeEnterpriseConfigurer.java

```diff
@@ -15,26 +15,44 @@ public class AerospikeEnterpriseConfigurer {
     private final AerospikeProperties aerospikeProperties;
     private final AerospikeEnterpriseProperties enterpriseProperties;
 
-    public void configure(GenericContainer<?> aerospikeContainer)  throws IOException, InterruptedException {
+    public void configure(GenericContainer<?> aerospikeContainer) throws IOException, InterruptedException {
         if (aerospikeProperties.getFeatureKey() == null || aerospikeProperties.getFeatureKey().isEmpty()) {
             log.warn("Evaluation feature key file not provided by 'embedded.aerospike.featureKey' property. " +
                     "Pay attention to license details: https://github.com/aerospike/aerospike-server.docker/blob/master/enterprise/ENTERPRISE_LICENSE");
         }
-
+        switchOffStopWritesSysMemoryPctMetric(aerospikeContainer);
         setupDisallowExpunge(aerospikeContainer);
     }
 
+    private void switchOffStopWritesSysMemoryPctMetric(GenericContainer<?> aerospikeContainer) throws IOException, InterruptedException {
+        log.info("Switching off 'stop-writes-sys-memory-pct'...");
+        String namespace = aerospikeProperties.getNamespace();
+        Container.ExecResult result = aerospikeContainer.execInContainer("asadm", "--enable", "-e",
+                String.format("manage config namespace %s param stop-writes-sys-memory-pct to 100", namespace));
+        logStdout(result);
+        if (result.getExitCode() != 0) {
```

Comment:
exit code check does not show much - let's try to match the correct output, if not - fail